### ISSUE TITLE
Combine zoom hashes into single hash using base64 encoding

### ISSUE THREAD

**Issue by jamessspanggg** [reposense/RepoSense#1195]
> After this PR, can you put up an issue to combine all the zoom hashes into a single hashes using base64 encoding?
This is to centralize the use of aliases, to limit the appearance of e.g. `zU` in our code

_Originally posted by @eugenepeh in [link](https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMjQzMzIzMjkxOnYy/pull_request_review_threads/discussion)_

With #830 merged, quite a number of zoom hashes are used to restore the zoom tab after reloading. Let's combine them to a single hash using base64 encoding.

**Comment by jamessspanggg** [comment]
@eugenepeh should we do the same for summary as well as authorship? the hash params for these views are increasing as well

**Comment by eugenepeh** [comment]
>@eugenepeh should we do the same for summary as well as authorship? the hash params for these views are increasing as well

Yes ideally all should be combined into a single base64 encoding, which would then be pre and post processed on every save/reload.
